docs/src/_concepts/project.md

```diff
@@ -359,3 +359,19 @@ The remaining tables in the database are used exclusively by Meltano UI, mostly
 - `oauth`
 - `embed_tokens`
 - `subscriptions`
+
+### Support for other database types
+
+Meltano currently supports the following databases as backends for state and configuration:
+
+* PostgreSQL
+* SQLite
+* MS SQL Server
+
+PostgresSQL and SQLite are supported out of the box, while the MS SQL Server requires installing Meltano with the [`mssql` extra](/guide/advanced-topics#installing-optional-components).
```

Comment:
while extra is technically right here, I think it could be confusing with native Meltano extras. Can we specify this is a Python extra?